src/Microsoft.DotNet.Cli.Utils/CommandResolution/ProjectToolsCommandResolver.cs

```diff
@@ -95,7 +95,7 @@ public CommandSpec Resolve(CommandResolverArguments commandResolverArguments)
         }
 
         private CommandSpec ResolveCommandSpecFromToolLibrary(
-            LibraryRange toolLibrary,
+            ProjectLibraryDependency toolLibrary,
```

Comment:
Why not still LibraryRange? NuGet seems to have a similar type.

Honestly should probably be https://github.com/NuGet/NuGet.Client/blob/3a0ee64c457cb9f4522da91812a48942a6122132/src/NuGet.Core/NuGet.LibraryModel/ToolDependency.cs

now that it's there.